metrics/lib/common.bash

```diff
@@ -24,6 +24,9 @@ KSM_SLEEP_FILE="${KSM_BASE}/sleep_millisecs"
 KSM_AGGRESIVE_PAGES=1000
 KSM_AGGRESIVE_SLEEP=50
 
+PKGDEFAULTCONFIG="${PKGDEFAULTCONFIG:-/usr/share/defaults/kata-containers/configuration.toml}"
```

Comment:
See comment around the usage point of this - I suspect we might/should be using `kata-env` to get this info.